mac-os-build/defaults.py

```diff
@@ -0,0 +1,36 @@
+# Copyright (C) 2018 VyOS maintainers and contributors
```

Comment:
This file is only required for the `check_build_config()` function, which is not used in this scenario.